pkg/agent/baker.go

```diff
@@ -63,8 +63,8 @@ func (t *TemplateGenerator) getWindowsNodeCustomDataJSONObject(config *datamodel
 	profile := config.AgentPoolProfile
 	//get parameters
 	parameters := getParameters(config, "", "")
-	//get variable cloudInit
-	variables := getCustomDataVariables(config)
+	//get variable custom data
+	variables := getCSECommandVariables(config)
```

Comment:
why do we use cseCommand variable in customData json?

Reply:
Windows customdata are mainly defined in CSEvaraibles, I can copy these variables to CustomdataVariables and use getCustomDataVariables here.